opentracing-spring-cloud-starter-zipkin/src/main/java/io/opentracing/contrib/spring/cloud/starter/zipkin/ZipkinAutoConfiguration.java

```diff
@@ -34,14 +39,12 @@
 import zipkin2.Span;
 import zipkin2.reporter.AsyncReporter;
 import zipkin2.reporter.Reporter;
-import zipkin2.reporter.Sender;
 import zipkin2.reporter.okhttp3.OkHttpSender;
 
 @Configuration
 @ConditionalOnClass(brave.opentracing.BraveTracer.class)
 @ConditionalOnMissingBean(io.opentracing.Tracer.class)
 @ConditionalOnProperty(value = "opentracing.zipkin.enabled", havingValue = "true", matchIfMissing = true)
-@AutoConfigureAfter(name = "me.snowdrop.opentracing.tracer.jaeger.JaegerAutoConfiguration")
```

Comment:
Was there any specific reason why this was removed?

Reply:
I actually wanted to ask why is it here? 

this artifact does not depend on snowdrop

Reply:
Then that was a mistake on part when I originally added the code.
What I meant to add was `@AutoConfigureAfter(name = "io.opentracing.contrib.spring.cloud.starter.jaeger.JaegerAutoConfiguration")`

which would make `JaegerAutoConfiguration` the default auto-configuration mechanism when both it and ZipkinAutoconfiguration are on the classpath.

If it's not too much trouble, could you add the line, but with the correct package name? Thanks

Reply:
I think it would be undefined behavior to use both starters. Maybe it's better to fail on ambiguous beans than prefer one bean than the other

Reply:
Maybe so, but I'm not sure how we could fail when both beans are present still keep auto-configuration working as we have it

Reply:
Anyway, I'll approve as it is. In the unlikely scenario that a user has a problem later on, we can easily address it but re-adding something like this